jdi-light/src/test/java/io/github/epam/properties/ElementSearchStrategyTests.java

```diff
@@ -0,0 +1,45 @@
+package io.github.epam.properties;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.security.InvalidParameterException;
+import java.util.Properties;
+
+import static com.epam.jdi.light.common.Property.ELEMENT_SEARCH_STRATEGY;
+import static com.epam.jdi.light.common.PropertyValidationUtils.validateProperties;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+public class ElementSearchStrategyTests {
+
+    @DataProvider
+    public static Object[] negativeData() {
+        return new Object[]{"strct", "s0ft", "visible", "any,single"};
```

Comment:
"any,single" must be valid. You can see how this options set up in WebSettings -> setSearchStrategy(String p).
1. It splits by coma: String[] params = paramString.split(",");
2. And then it gets rid of all spaces: String visibilityParam = params[0].trim();
Thus it doesn't matter whether user use spaces in between words or don't. Both options are legal.

Put "any,single" in positive data after changing the regexp and add empty string "" to negative data

Reply:
Done